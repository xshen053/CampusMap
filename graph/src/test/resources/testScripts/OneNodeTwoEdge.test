# Create a graph
CreateGraph ONTE_graph

# Add a node
AddNode ONTE_graph A

# Add two edge
AddEdge ONTE_graph A A e1
AddEdge ONTE_graph A A e2

# Print the nodes in the graph and the outgoing edges from A
ListNodes ONTE_graph
ListChildren ONTE_graph A