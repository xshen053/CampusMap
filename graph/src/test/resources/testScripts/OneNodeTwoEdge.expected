# Create a graph
created graph ONTE_graph

# Add a node
added node A to ONTE_graph

# Add two edge
added edge e1 from A to A in ONTE_graph
added edge e2 from A to A in ONTE_graph

# Print the nodes in the graph and the outgoing edges from A
ONTE_graph contains: A
the children of A in ONTE_graph are: A(e1) A(e2)